app/views/overview/_list-row-actions.html

```diff
@@ -15,7 +15,15 @@
               <a href="" ng-click="row.startBuild(row.buildConfigs[0])">Start Build</a>
             </li>
             <li ng-if="'deploymentconfigs/instantiate' | canI : 'create'" role="menuitem">
-              <a href="" ng-click="row.startDeployment()">Deploy</a>
+              <a
+                href=""
+                ng-if="row.canDeploy()"
+                ng-click="row.startDeployment()">Deploy</a>
+              <a
+                href=""
+                ng-if="!(row.canDeploy())"
+                ng-attr-aria-disabled="{{row.canDeploy() ? undefined : 'true'}}"
+                ng-class="{ 'disabled-link': !row.canDeploy() }">Deploy (paused)</a>
```

Comment:
> I can wrap the (paused) with a span and ng-if="isPaused()"

Seems reasonable. Should probably be `(Paused)` with a capital.

> Also... should the above switch be updated? I was considering updating the switch to DeploymentConfig || Deployment, but not sure if that is right.

We don't support manual rollouts for upstream deployments in the web console. @jwforres switched to DeploymentRequest for DeploymentConfigs (#1434)... I suspect the API for upstream deployments is similar, although I haven't looked at it. We probably should add it, but not for this PR.

>  Do we want a Resume Rollouts button added here?

I think the link in the notification is good enough, and what you have currently is consistent with what we do in the deployment configs page.